flake.nix

```diff
@@ -69,6 +69,27 @@
                   );
             };
           };
+          exampleRagePlugin = buildRustPackage rec {
```

Comment:
Would overriding `pkgs.rage` be an option? Or at least using the `src` from the `rage` package. That way the version of `rage` we use throughout the checks would be aligned.

Reply:
Nice idea, seems to work fine with `overrideAttrs`